### PR TITLE
Use gppkg v2 to pack on GPDB7

### PR DESCRIPTION
**Pull request by beeender**
- Use new gppkg for GPDB7.
- Use registry-image over docker-image to save costs.
- Adapt resources for the GPDB7 binary release candidates.
- Bump version to 2.2.1.

### CODE REVIEW

**Review by yihong0618 — APPROVED** [review]
LGTM

File: concourse/scripts/test_plcontainer.sh [hunk 1]
```diff
@@ -2,11 +2,27 @@
 
 set -exo pipefail
 
+function _install_gppkg() {
+    if [[ ${GP_MAJOR_VERSION} == "7" ]]; then
+        "/home/gpadmin/bin_gppkg_v2/gppkg" install -a ./*.gppkg
```
[the remark this answers is not in the document]
Comment (beeender) [reply concourse/scripts/test_plcontainer.sh:7]: `*.gppkg` will be extracted by bash I think.

[the remark this answers is not in the document]
Comment (Sasasu) [reply concourse/scripts/test_plcontainer.sh:7]: `./foo.gppkg` to `/<absolute path>/foo.gppkg`

**Comment by stkim1** [comment]
LGTM. Sent an e-mail for you to review.

**Comment by beeender** [comment]
@stkim1 
HiaHiaHiaHia!!! Glad to hear from you!! I just found your email in the gmail junk folder XD

**Pull request merged**